### ISSUE TITLE
Missing cache file in dist

### ISSUE THREAD

**Issue by Kosai106** [dequelabs/axe-core-npm#519]
```
ERROR in ./node_modules/@axe-core/react/dist/index.js
Module not found: Error: Can't resolve './cache' in '/Users/**/**/**/node_modules/@axe-core/react/dist'
 @ ./node_modules/@axe-core/react/dist/index.js 20:30-48
 @ ./src/index.tsx
```

**Product:** `react`

**Expectation:**
I expect the file to be present so my application will run.

**Actual:**
The file is missing from the `node_modules` meaning my app is not able to compile and start.

**Motivation:**
`4.4.0` works but doesn't support React 17
`4.4.2` doesn't work because of this issue https://github.com/dequelabs/axe-core-npm/issues/479, but solves React 17 support...

Hence why we use `@axe-core/react@next` 🙃

```bash
axe-core version: 4.4.1 # This is defined in your own package
@axe-core/react: 4.4.3-alpha.288

- Node version: 16.13.0
- Platform: MacOS Monterey
```

**Comment by michael-siek** [comment]
Hey @Kosai106 Thank you very much for the contribution! this was released in 4.4.3.

**Comment by padmavemulapati** [comment]
cache files observed and not finding error with ../react/dist/index.js. 
Tests are working as expected
<img width="288" alt="image" src="https://user-images.githubusercontent.com/47409579/184813522-aba1ed4f-d04d-4568-91fe-3f1411c3815d.png">
